timbermill-client/src/main/java/com/datorama/oss/timbermill/unit/EventsList.java

```diff
@@ -0,0 +1,17 @@
+package com.datorama.oss.timbermill.unit;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class EventsList extends ArrayList<Event> {
```

Comment:
How does this look as a String?

Reply:
by calling ObjectMapper().writeValueAsString() on some EventsList,  got this:

[{"@type":"SpotEvent","taskId":"metadata_timbermill_client_heartbeat___e4510321_da9e_4eea_a8da_c10f2661817a_timbermill2","time":"2020-09-09T12:59:49.559Z","name":"metadata_timbermill_client_heartbeat","strings":{"threadName":"timbermill-heartbeat-thread"},"text":{},"context":{},"metrics":{"submitAmount":0,"avgSubmitDuration":0.0,"outputBufferSize":0,"maxSubmitDuration":0},"logs":[],"env":"test","status":"SUCCESS"},{"@type":"StartEvent","taskId":"Event___52c0fd59_b85a_41a7_aaf0_67b819a43979_timbermill2","time":"2020-09-09T12:59:49.56Z","name":"Event","strings":{"threadName":"main"},"text":{},"context":{},"metrics":{},"logs":[],"env":"test"},{"@type":"InfoEvent","taskId":"Event___52c0fd59_b85a_41a7_aaf0_67b819a43979_timbermill2","time":"2020-09-09T12:59:49.56Z","strings":{},"text":{},"context":{"context1":"context1"},"metrics":{},"logs":[],"env":"test"},{"@type":"StartEvent","taskId":"Event2___8bfb79e9_ad0c_457e_a206_5ea2e6a610b9_timbermill2","time":"2020-09-09T12:59:49.561Z","name":"Event2","parentId":"Event___52c0fd59_b85a_41a7_aaf0_67b819a43979_timbermill2","strings":{"threadName":"main"},"text":{},"context":{},"metrics":{},"logs":[],"env":"test"},{"@type":"InfoEvent","taskId":"Event2___8bfb79e9_ad0c_457e_a206_5ea2e6a610b9_timbermill2","time":"2020-09-09T12:59:49.561Z","strings":{},"text":{},"context":{"context2":"context2"},"metrics":{},"logs":[],"env":"test"},{"@type":"StartEvent","taskId":"Event3___ecd14bb2_041c_41fb_8e1d_366ac5f40a92_timbermill2","time":"2020-09-09T12:59:49.563Z","name":"Event3","parentId":"Event2___8bfb79e9_ad0c_457e_a206_5ea2e6a610b9_timbermill2","strings":{"threadName":"Thread-2"},"text":{},"context":{},"metrics":{},"logs":[],"env":"test"},{"@type":"SuccessEvent","taskId":"Event3___ecd14bb2_041c_41fb_8e1d_366ac5f40a92_timbermill2","time":"2020-09-09T12:59:49.563Z","strings":{},"text":{},"context":{},"metrics":{},"logs":[],"env":"test"},{"@type":"StartEvent","taskId":"Spot___107fbf74_0695_465d_844f_0bbfc088dde5_timbermill2","time":"2020-09-09T12:59:49.563Z","name":"Spot","parentId":"Event___52c0fd59_b85a_41a7_aaf0_67b819a43979_timbermill2","strings":{"threadName":"Thread-3"},"text":{},"context":{},"metrics":{},"logs":[],"env":"test"},{"@type":"SuccessEvent","taskId":"Spot___107fbf74_0695_465d_844f_0bbfc088dde5_timbermill2","time":"2020-09-09T12:59:49.563Z","strings":{},"text":{},"context":{},"metrics":{},"logs":[],"env":"test"},{"@type":"SuccessEvent","taskId":"Event2___8bfb79e9_ad0c_457e_a206_5ea2e6a610b9_timbermill2","time":"2020-09-09T12:59:50.563Z","strings":{},"text":{},"context":{},"metrics":{},"logs":[],"env":"test"},{"@type":"SuccessEvent","taskId":"Event___52c0fd59_b85a_41a7_aaf0_67b819a43979_timbermill2","time":"2020-09-09T12:59:50.563Z","strings":{},"text":{},"context":{},"metrics":{},"logs":[],"env":"test"}]

Reply:
Good enough